src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/templates/basic-af/.content.xml

```diff
@@ -4,6 +4,7 @@
     <jcr:content
         author="adobe"        
         cq:templateType="/conf/${appId}/settings/wcm/template-types/af-page"
+        jcr:description="This template offers a layout with only headers and footers for developing adaptive forms using Foundation Components."
```

Comment:
`With only header and footer` means no content. Doesn't it describe this template equivalent to blank or kind of blank.